ecs-logs-collector.sh

```diff
@@ -584,6 +585,24 @@ enable_ecs_agent_debug() {
   esac
 }
 
+get_docker_goroutines() {
+  try "dump docker goroutine stacks"
+  dstdir="${info_system}/docker_stacks"
+  mkdir -p ${dstdir}
```

Comment:
We're driving shellcheck fixup and enforcement PRs, so let's proactively get these tidied up and quoted:

```suggestion
  mkdir -p "${dstdir}"
```

Reply:
Added.